.gitignore

```diff
@@ -1,12 +1,12 @@
-.env
+
 *.pyc
-*#
-*~
-*.swp
-#*
-local_settings.py
-slackbot_settings.py
-/build
-/dist
-/*.egg-info
-.cache
+
+.idea/encodings.xml
+
+.idea/misc.xml
+
+.idea/modules.xml
+
+.idea/snakeman.iml
+
+.idea/workspace.xml
```

Comment:
@migetman9 we should restore the original stuff in this file, and just exclude the .idea directory instead of the individual things in it